src/ex2/PokemonClient.js

```diff
@@ -0,0 +1,32 @@
+export class PokemonClient {
```

Comment:
Why do you need two methods in order to fetch pokemons? you can use only the fetchPokemons method and remove fetchSinglePokemon.
this way you don't have logic duplication and you can just send fetchPokemons an array that contains only one id.

Reply:
fixed!